src/client/Client.cc

```diff
@@ -3711,8 +3731,9 @@ int Client::mark_caps_flushing(Inode *in)
   int flushing = in->dirty_caps;
   assert(flushing);
 
-  if (flushing && !in->flushing_caps) {
+  if (!in->flushing_caps) {
```

Comment:
You're just clearing out the flushing check here because we already asserted it above, or was there some other reason?